.jscsrc

```diff
@@ -0,0 +1,4 @@
+{
+    "esnext": true,
+    "preset": "airbnb",
```

Comment:
The trailing comma is currently breaking JSCS when you try running it from command line: `./node_modules/jscs/bin/jscs src/`